foundry/collections/entityCollections/users.d.ts

```diff
@@ -35,4 +35,10 @@ declare class Users extends EntityCollection<User> {
    * @param activityData - The object of activity data
    */
   protected static _handleUserActivity(userId: string, activityData: User.ActivityData): void;
+
+  /* -------------------------------------------- */
+  /*  Constants              */
+  /* -------------------------------------------- */
```

Comment:
We normally strip these kinds of comments out by now. There is no need to add it.